be/src/storage/rowset/column_reader.h

```diff
@@ -101,6 +100,13 @@ class ColumnReader {
 
     ~ColumnReader();
 
+    // Disable copy and assignment
```

Comment:
I remember there is some macro in google library that can disable copy & assignment function.